app/models/theme.rb

```diff
@@ -151,6 +154,23 @@ def default?
     SiteSetting.default_theme_id == id
   end
 
+  def enabled?
+    enabled = true
+
+    minimum_version = remote_theme&.minimum_discourse_version
+    maximum_version = remote_theme&.maximum_discourse_version
+
+    if minimum_version
+      enabled = false unless Discourse.has_needed_version?(Discourse::VERSION::STRING, minimum_version)
```

Comment:
If you're not depending on `enabled` later it would make more sense to `return false` and avoid any further checks. You could probably avoid the `enabled` variable altogether by returning `true` if you make it until the end of the method.